package-lock.json

```diff
@@ -14662,710 +10967,278 @@
         "selenium-webdriver": "^3.0.1"
       }
     },
-    "webidl-conversions": {
-      "version": "6.1.0",
-      "resolved": "https://registry.npmjs.org/webidl-conversions/-/webidl-conversions-6.1.0.tgz",
-      "integrity": "sha512-qBIvFLGiBpLjfwmYAaHPXsn+ho5xZnGvyGvsarywGNc8VyQJUMHJ8OBKGGrPER0okBeMDaan4mNBlgBROxuI8w==",
-      "dev": true
-    },
     "webpack": {
```

Comment:
:exclamation:**Cycode**: Security vulnerability found in newly introduced dependency. 

||| 
 |---|---|
|**Severity**|High|
|**Issue**|Cross-realm object access in Webpack 5: CVE-2023-28154|
|**Ecosystem**|NPM|
|**Dependency**|`webpack`|
|**Dependency Paths**|`@angular-devkit/build-angular 15.0.0 -> webpack 5.75.0`|
|**Direct Dependency**|No|
|**Development Dependency**|Yes|
|**Upgrade**|`5.76.0`|

 
 Webpack 5 before 5.76.0 does not avoid cross-realm object access. ImportParserPlugin.js mishandles the magic comment feature. An attacker who controls a property of an untrusted object can obtain access to the real global object.
 <!-- b7812104-fd85-498b-bfb8-a857aea307ba -->